Watchman.Discord/Program.cs

```diff
@@ -17,7 +18,18 @@ public static async Task Main(string[] args)
             var configuration = JsonConvert.DeserializeObject<DiscordConfiguration>(File.ReadAllText(configPath));
             
             var watchman = new WatchmanBot(configuration);
-            await watchman.GetWorkflowBuilder().Run();
+            var workflowBuilder = watchman.GetWorkflowBuilder();
+            while (true)
```

Comment:
czemu while true?
nie lepiej dać mu kilka prób? np 3 albo 5 

co jeśli przez głupi błąd zrobi spam discordowi i wykryje go jako bota spamującego?

Reply:
Dam może jakiś kilku sekundowy Delay + sprawdzanie czy w ostatnich kilku mintach nie wywaliło kilka razy
Głównie chodziło mi o to, żeby w razie wyjątków nie wyrzucało całego bota, a żeby łapał te nie złapane wyjątki i uruchamiał bota na nowo
Ale faktycznie warto zabezpieczyć to w razie gdyby wyjątek był podczas budowania bota, bo wtedy byłaby nieskończona pętla i 100% użycia procesora

Reply:
co z tą pętlą?

Reply:
W sensie? To już jest poprawione

Reply:
aaa bo ty tak to zrobiłeś
ja bym zrobił tego ifa if (numberOfExceptionsInLastTime++ > HOW_MANY_EXCEPTIONS_IN_SHORT_TIME_TO_STOP_BOT) 
w środku while()

Reply:
A przenieść do heada pętli?

Reply:
w sumie możesz